src/sass/modules/_m-modal.scss

```diff
@@ -11,6 +11,41 @@
   top: 0;
   width: 100%;
   z-index: 3;
+  cursor: pointer;
+
+  &-split {
+    .va-modal-inner {
+      min-width: 710px;
+    }
+    &-container {
+      display: flex;
+    }
+    &-left,
+    &-right {
+      flex: 1;
+      padding: 0 20px;
+    }
+    p:first-child,
```

Comment:
Could we style a class instead of `p`, `h5`, and `ul` elements? Or use `>` to make the rules narrower, if that makes more sense? 

I keep getting tripped up by this in other places, where someone has styled elements for a different context and I have to overwrite those styles with `!important` or more specific ones.